docs/en/latest/plugins/http-dubbo.md

```diff
@@ -0,0 +1,127 @@
+---
+title: http-dubbo
+keywords:
+  - Apache APISIX
+  - API 网关
+  - Plugin
+  - http-dubbo
+  - http to dubbo
+description: This document contains information about the Apache APISIX http-dubbo Plugin.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+## Description
+
+The `http-dubbo` plugin can encode HTTP requests into Dubbo protocol and forward them to upstream services (Note: in
```

Comment:
The plugin also transcode back to HTTP JSON right? If so, please add this info as well. For example, you can say "transcode between X and X".